files/zh-cn/web/api/abortcontroller/signal/index.md

```diff
@@ -0,0 +1,71 @@
+---
+title: AbortController.signal
+slug: Web/API/AbortController/signal
+page-type: web-api-instance-property
+tags:
+  - API
+  - AbortController
+  - Experimental
+  - Fetch
+  - Property
+  - Reference
+  - signal
+translation_of: Web/API/AbortController/signal
+---
+{{APIRef("DOM")}}
+
+{{domxref("AbortController")}} 接口的只读属性 **`signal`** 返回一个 {{domxref("AbortSignal")}} 实例对象，该对象可以根据需要处理 DOM 请求通信，既可以建立通信，也可以终止通信。
+
+## 值
+
+一个 {{domxref("AbortSignal")}} 实例对象。
+
+## 示例
+
+在下面的这段代码中，我们将通过 [Fetch API](/zh-CN/docs/Web/API/Fetch_API) 来下载一段视频。
+
+首先通过 {{domxref("AbortController.AbortController","AbortController()")}} 构造函数来创建一个 controller 实例，然后通过 {{domxref("AbortController.signal")}} 属性获取到它的关联对象{{domxref("AbortSignal")}} 的引用。
+
+当 [fetch request](/zh-CN/docs/Web/API/fetch) 初始化后，将 `AbortSignal` 作为一个选项传入请求的选项参数中（如下 `{signal}`）。这将 signal，controller 与 fetch 请求关联起来，允许我们通过调用 {{domxref("AbortController.abort()")}} 来取消 fetch 请求，正如下第二个事件监听器所示。
```

Comment:
```suggestion
当 [fetch request](/zh-CN/docs/Web/API/fetch) 初始化后，将 `AbortSignal` 作为一个选项传入请求的 options 对象中（如下 `{signal}`）。这将 signal 和 controller 与 fetch 请求关联起来，允许我们通过调用 {{domxref("AbortController.abort()")}} 来取消 fetch 请求，正如下第二个事件监听器所示。
```

Reply:
The `options` should not be translated, it's the param's name <https://developer.mozilla.org/en-US/docs/Web/API/fetch#parameters>